newsfragments/513.changed.md

```diff
@@ -0,0 +1 @@
+Synapse: Automatically configure MSC3266, MSC4222 and set duration and rate limits according to MSC4140 when matrixRTC is enabled.
```

Comment:
```suggestion
Configure Synapse appropriately for Element Call when matrixRTC is enabled.
```

I don't think we need to go into too much detail on the specifics, the outcome is what we care about